PxCs/Extensions/SoundExtension.cs

```diff
@@ -0,0 +1,302 @@
+using System;
+using System.Text;
+using System.IO;
+using System.Collections.Generic;
+
+namespace PxCs.Extensions
```

Comment:
This namespace is used for ObjectType extensions only. i.e. IntPtr.MarshalAsString().

How about using another one like Sound/Audio/...?